.github/workflows/element-with-modules.yml

```diff
@@ -0,0 +1,328 @@
+name: Element with modules
+
+on:
+  push:
+    branches: # we want to build the widgets from the push into Element staging and publish a main container
+      - main
+    tags: # we want to build the stable widgets into stable Element and publish the container
+      - '@nordeck/element*module*'
+  pull_request: # we want to build the widgets from the PR into Element develop & staging
+  schedule: # we want to build the stable widgets into stable Element and publish the container, plus a verification of all dev versions
+    - cron: '0 5 * * 1-6'
+  dispatch: # we build whatever the user inputs and publish the container
+    inputs:
+      element:
+        description: 'Element Web version (git ref)'
+        required: true
+        type: 'string'
+      guestModule:
+        description: 'Element Guest module version (release tag/git ref/none)'
+        required: true
```

Comment:
If it can be none, why is is then required?

Reply:
required that you specifically say you want `none`. i can build something with a default if you prefer.

Reply:
> required that you specifically say you want none.

:+1: